plume-front/src/editor.rs

```diff
@@ -62,7 +64,121 @@ impl From<stdweb::private::ConversionError> for EditorError {
         EditorError::TypeError
     }
 }
-
+#[derive(Serialize, Deserialize)]
+struct AutosaveInformation {
+    contents: String,
+    title: String,
+    last_saved: f64,
+}
```

Comment:
while content is probably the most annoying to loose, if we do a local save, we should as well save everything (including subtitle, tags...)

Reply:
True enough, easy enough to add too